roles/core/conman/molecule/default/verify.yml

```diff
@@ -0,0 +1,101 @@
+---
+- name: Verify
+  hosts: management1
+  tasks:
+
+    - name: Collect package facts
+      package_facts:
+        manager: auto
+
+    - name: Collect services facts
+      service_facts:
+
+    - name: Assert common packages are installed
+      assert:
+        that: "'{{ item }}' in ansible_facts.packages"
+      loop:
+        - conman
+        - ipmitool
+      when:
+        - ansible_facts.os_family == "RedHat"
+
+    - name: Assert needed el7 conman packages are installed
+      assert:
+        that: "'{{ item }}' in ansible_facts.packages"
+      loop:
+        - libselinux-python
+        - policycoreutils-python
+      when:
+        - ansible_facts.os_family == "RedHat"
+        - ansible_facts.distribution_major_version == "7"
+
+    - name: Assert needed el8 conman packages are installed
+      assert:
+        that: "'{{ item }}' in ansible_facts.packages"
+      loop:
+        - python3-libselinux
+        - python3-policycoreutils
+      when:
+        - ansible_facts.os_family == "RedHat"
+        - ansible_facts.distribution_major_version == "8"
+
+    - name: Retrieve file /etc/conman.conf system status
+      stat:
+        path: /etc/conman.conf
+      register: reg_file_conman
+      changed_when: false
+
+    - name: Assert file /etc/conman.conf exists
+      assert:
+        that:
+          - reg_file_conman.stat.exists
+          - reg_file_conman.stat.mode == '0644'
+          - reg_file_conman.stat.pw_name == 'root'
+          - reg_file_conman.stat.gr_name == 'root'
+
+    - name: Retrieve client configuration in /etc/conman.conf file
+      lineinfile:
+        path: /etc/conman.conf
+        regexp: 'console name="compute1" dev="ipmitool.exp 10.10.102.1 ADMIN ADMIN"'
```

Comment:
There is no need to create a fake host, you can check the line below:
`console name="instance" dev="ipmitool.exp 10.10.102.1 ADMIN ADMIN"`

Reply:
I just added a fake compute host to be closer of a real installation/configuration.
IMHO, having a client configuration is helpful to read the molecule test suite and to understand it better.

Reply:
There is no such thing as a real installation/configuration. Some will use baremetal management nodes, other will use containers or VMs.
The purpose of this unit test it to test the conman role. The role needs a host in the inventory to create a conman.conf file. `instance` is an host, and it is added to the configuration file during the run of the play. We don't need more than that today.

Reply:
Blah... I will modify this for you. Thanks.